azure-cli/README.md

```diff
@@ -0,0 +1,29 @@
+# GitHub Action for the Azure CLI
+
+The GitHub Action for [Azure CLI](https://github.com/Azure/azure-cli) wraps the Azure CLI to enable managing Azure resources inside of an Action.
+
+To log into a Azure, we recommend using the [Azure Login](../azure-login) Action.
+
+## Usage
+
+```
+
+ action "Azure CLI" {
+  uses = "Azure/github-actions/cli@master"
+  env = {
+    AZURE_SCRIPT_PATH = "<Relative path in your repository>"
+    AZURE_SCRIPT = "az account list"
```

Comment:
Can you mention that this is <inline script> and then give example ?

Reply:
done ! thanks.